tests/test_environment_AngularSeparation.py

```diff
@@ -1,61 +1,74 @@
 import numpy.testing as npt
 import numpy as np
 import freud
-import rowan
 import unittest
 from util import make_box_and_random_points
 
 
 class TestAngularSeparation(unittest.TestCase):
-    def test_getN(self):
+    def test_getN_global(self):
```

Comment:
This should probably be `TestAngularSeparationGlobal.test_getN` instead of `TestAngularSeparation.test_getN_global`.